src/map/clif.cpp

```diff
@@ -3827,7 +3827,7 @@ RecvResult clif_parse_GlobalMessage(Session *s, dumb_ptr<map_session_data> sd)
         /* It's not a spell/magic message, so send the message to others. */
         Packet_Head<0x008d> head_8d;
         head_8d.block_id = sd->bl_id;
-        XString repeat_8d = mbuf;
+        XString repeat_8d = repeat;
```

Comment:
@wushin using `repeat` here is bad because it comes directly from the client so it can be spoofed.
<br>
You should use 
```c++
XString repeat_8d = STRPRINTF("%s : %s"_fmt, sd->status_key.name, mbuf);
```